### PR TITLE
New programs added

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#76 #75 
#### What does this implement/fix? Explain your changes.
I added two programs (`52_17` and `112_26`) similar to `pomodoro`.

#### Any other comments?
I believe the implementation is truly suboptimal, but since I started using `mytimer` in my day-to-day schedule and wanted to have those other programs I decided to add them both in a PR. I believe we can enhance implementation quality by refactoring later.

### CODE REVIEW

File: mytimer/params.py [hunk 1]
```diff
@@ -163,6 +163,18 @@
         "second": 0,
         "message": "Time to work (25 mins)",
     },
+    "52_17": {
```
Comment (sepandhaghighi) [inline mytimer/params.py:166]: "We never used `_` in the program name.
I suggest `52/17` or `52-17`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 🔥 
Please take a look at my comments 💯

File: mytimer/params.py (hunk 1 above)
Comment (sadrasabouri) [reply mytimer/params.py:166]: You are right. I was just thinking about potential problems using `-` or `/` would make when users input the program name in CLI.

Comment (sadrasabouri) [reply mytimer/params.py:166]: I tested it and using `-` is pretty fine (at least in Linux terminal), please test it on Windows, too.
I will double-check in macOS terminal before merging.

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
Would you please update `README.md` references section:

5. [The Rule of 52 and 17: It's Random, But it Ups Your Productivity](https://www.themuse.com/advice/the-rule-of-52-and-17-its-random-but-it-ups-your-productivity)

6. [Desktime’s Productivity Research: An Overview of Our Finds Throughout the Years](https://desktime.com/blog/productivity-research)

**Review by sepandhaghighi — APPROVED** [review]
@sadrasabouri Well done 🔥

**Pull request merged**